bin/restic_backup.sh

```diff
@@ -33,17 +33,36 @@ assert_envvars() {
 		fi
 	done
 }
+
+warn_on_missing_envvars() {
+	unset_envs=
```

Comment:
Could make this variable `local`

Reply:
Done, good point. Missing bash experience shows..